### PR TITLE
Arweave read write flow

### PR DESCRIPTION
**Pull request by edisonz0718**
#122

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,6 +16,8 @@ DOCKER_BUILD=docker build --cache-from
 
 DEVELOPMENT_DB_URI = postgresql://postgres:@api-database:5432/postgres?sslmode=disable
 
+GOOGLEAPIS_DIR=./proto/third_party/googleapis
```
Comment (LeoHChen) [inline Makefile:19]: do we have to include the source code of third party library into our repo?  can we do a dynamic check out and build?

Comment (LeoHChen) [reply Makefile:19]: If we must include 3rd party source code, what's the version? And what is the source url?

**Review by LeoHChen — COMMENTED** [review]
what's our strategy to deprecate the v1 api?  shall we create a v1 release branch to handle v1 api only?

File: Makefile (hunk 1 above)
Comment (edisonz0718) [reply Makefile]: The license, version and source url were in the same folder as the proto definitions. But I removed the 3rd party code as we actually don't depend on them after testing it again.

**Comment by edisonz0718** [comment]
> what's our strategy to deprecate the v1 api? shall we create a v1 release branch to handle v1 api only?

v1 api is for FO. So unless we want to migrate FO to v2 or deprecate FO, we want to run both versions.

**Review by LeoHChen — APPROVED** [review]

**Pull request merged**